x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/session/Sequence.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.eql.session;
+
+import org.elasticsearch.common.collect.Tuple;
+import org.elasticsearch.search.SearchHit;
+
+import java.util.List;
+
+public class Sequence {
+
+    private final List<Tuple<Object, List<SearchHit>>> events;
```

Comment:
at some point, we'll also add join_keys and the other fields mentioned in the initial issue, but this is already good enough as a placeholder until then

Reply:
Right - the response will be fleshed out. For sequences I've added the join keys in the tuple under object (the response json is an array but the examples contain only one key).